iis-wf/iis-wf-affmatching/src/main/java/eu/dnetlib/iis/wf/affmatching/match/AffOrgMatcher.java

```diff
@@ -44,16 +45,18 @@
      * <li>choosing the best matching affiliation-organization pair for each affiliation, done by {@link BestAffMatchResultPicker}
      * </ul> 
      */
-    public JavaRDD<AffMatchResult> match(JavaRDD<AffMatchAffiliation> affiliations, JavaRDD<AffMatchOrganization> organizations) {
+    public JavaRDD<AffMatchResult> match(JavaRDD<AffMatchAffiliation> affiliations, JavaRDD<AffMatchOrganization> organizations, 
+            JavaRDD<AffMatchDocumentOrganization> documentOrganizations) {
```

Comment:
I think it might be confusing for someone reading we are looking for affiliation organization relations while providing `documentOrganizations` at the same time. It will be worth explaining what is the purpose of the latter.